app/modules/asset_groups/metadata.py

```diff
@@ -60,7 +60,7 @@ def __init__(self, request_json):
 
     @classmethod
     def _validate_field_type(cls, dictionary, field, field_type, error_str):
-        if field_type == uuid.UUID and dictionary[field]:
+        if field_type == uuid.UUID and dictionary[field] is not None:
```

Comment:
The change I made was wrong but this is also wrong. The FE sends fields as None so we need to accept them so what this should be is 
if field_type == uuid.UUID and dictionary[field] and dictionary[field] is not None:
I'll do that today and correct any tests that blow up